tensorflow/lite/micro/micro_profiler.cc

```diff
@@ -49,7 +49,7 @@ uint32_t MicroProfiler::GetTotalTicks() const {
 }
 
 void MicroProfiler::Log() const {
-#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+#ifdef TF_LITE_DISABLE_PROFILING
```

Comment:
- I think it's supposed to be `ifndef` instead of `ifdef` ?
- Note that this code is never called in our setups because we have our own 'print results' function that prints a table and some json output. Is the intention of this change to reduce codesize?

Reply:
thanks!

Reply:
btw, the intention was to follow what's already there - just making the profiling slightly more independent of the error strings:

```
 void MicroProfiler::LogTicksPerTagCsv() {
-#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+#ifndef TF_LITE_DISABLE_PROFILING
[....]
```